example/ck_tile/17_grouped_gemm/grouped_gemm.hpp

```diff
@@ -46,6 +46,8 @@ auto create_args(int argc, char* argv[])
     return std::make_tuple(result, arg_parser);
 }
 
+std::size_t GetWorkspaceSize(const std::vector<grouped_gemm_kargs>& gemm_descs);
+
 float grouped_gemm_calc(const std::vector<grouped_gemm_kargs>& gemm_descs,
```

Comment:
You have renamed this function in `*.cpp` file https://github.com/ROCm/composable_kernel/pull/1713/files/0f0f0b325ba3d87abbdb250d0d6df9c2a9441000..ff8dfb4b405ebca423db6b7a46e7ce23a060a71a#diff-0232f5d7482b16236c50bcb95109f0ce637604309f4f5ba64b176f1991f9b503R111